vmdb/app/models/metric/processing.rb

```diff
@@ -38,6 +47,10 @@ def self.process_derived_columns(obj, attrs, ts = nil)
           result[col] = ( attrs[:cpu_usage_rate_average]     / 100 * total_cpu ) unless (total_cpu == 0 || attrs[:cpu_usage_rate_average].nil?)
         elsif group == "memory"
           result[col] = ( attrs[:mem_usage_absolute_average] / 100 * total_mem ) unless (total_mem == 0  || attrs[:mem_usage_absolute_average].nil?)
+        elsif group == "logicalcpus"
```

Comment:
Jason and I found out that the `when "used" if "cpu"` part of this case block is never actually called because there's no derived field called `derived_cpu_used`.  And, as Jason points out above, the amount of "used" CPU doesn't really make sense here.

The same can be said for "used" "logicalcpus".

I think the whole PR can be reduced to changing the inline `if` on https://github.com/ManageIQ/manageiq/pull/2701/files#diff-e9361526ee8f29e2fb33f569b717ffa1R65 to allow for `VmOrTemplate` and `Container`.  Then, updating the `capture_numvcpus` method in `vim_performance_state` to handle looking up logical cpus from Container Nodes.